maquetado/scss/detail.scss

```diff
@@ -0,0 +1,11 @@
+.field {
+  font-size: 19px;
+}
+
+.field-description {
+  font-size: 19px;
+}
+
+.title {
+  font-size: 2em;
```

Comment:
use only px or em for the font-sizes. we normally use px

Reply:
So I should change this?

Reply:
yes, please :D

Reply:
My doubt was because you told me you use both px or em :D

Reply:
I tried to say that in a project you should only use one of the two. If you use px you have to use it for every text class. If you mix px and ems or rems you will have weird changes when using media queries. https://medium.com/code-better/css-units-for-font-size-px-em-rem-79f7e592bb97

Reply:
Now I get it, great!